### PR TITLE
[PM-1541] Adjust the dialogs max-width

### PR DESCRIPTION
**Pull request by Hinton**
## Type of change

```
- [x] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Change dialog max width to w-xl for default and w-3xl for large.

## Screenshots

These dialogs had their width adjusted to better fit the content.

image

image

image

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

**Comment by Hinton** [comment]
I can do one better:

Before: https://components.bitwarden.com/?path=/story/component-library-dialogs-dialog--default
After https://62a88a6de5b807fa98886113-ublduyjzep.chromatic.com/?path=/story/component-library-dialogs-dialog--default

You can toggle between the different sizes. Do note that it's only resizing if you use the "service" story.

**Review by danielleflinn — APPROVED** [review]

**Review by danielleflinn — APPROVED** [review]
I agree with the sentiment in Slack that we should update the size of the more complex dialogs to `large`. So as long as that happens, these changes seem good to merge.

File: bitwarden_license/bit-web/src/app/secrets-manager/settings/dialog/sm-import-error-dialog.component.html [hunk 1]
```diff
@@ -13,7 +13,9 @@
       </ng-container>
       <ng-template body>
         <tr bitRow *ngFor="let line of errorLines">
-          <td bitCell class="tw-break-all">[{{ line.id }}] [{{ line.type }}] {{ line.key }}</td>
+          <td bitCell class="tw-whitespace-nowrap">
```
Comment (MGibson1) [inline bitwarden_license/bit-web/src/app/secrets-manager/settings/dialog/sm-import-error-dialog.component.html:16]: `tw-break-all` and `tw-whitespace-nowrap` aren't equivalent -- I kind of think they're roughly the opposite.

What's the purpose of this change and is there impact anywhere? IIRC this will cause modals that have too-long lines to create horizontal scroll bars, which sounds less-than-ideal

Comment (Hinton) [reply bitwarden_license/bit-web/src/app/secrets-manager/settings/dialog/sm-import-error-dialog.component.html:16]: They aren't, and that was an intentional change. I wanted the import name to be on a single line without weird breaks in the middle since it should be a fairly short string of `[id] Type`.

@bitwarden/team-secrets-manager-dev feel free to object to this change but seemed a bit better than setting a width.

Comment (Thomas-Avery) [reply bitwarden_license/bit-web/src/app/secrets-manager/settings/dialog/sm-import-error-dialog.component.html:16]: This is fine. 

`tw-break-all` wasn't really doing anything for us and with the dialog sizing changes this specific dialog
, `sm-import-error-dialog`, wasn't displaying as intended. 

### Before change 
before fix


### After change `tw-whitespace-nowrap`
after fix

**Review by Thomas-Avery — APPROVED** [review]

**Review by MGibson1 — APPROVED** [review]
